recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+channel_sources:
+  - conda-forge/label/cython_dev,conda-forge,defaults
```

Comment:
Can we remove defaults here? We don't use it anymore in conda-forge. I think...

Reply:
Yep. Here's the relevant PR ( https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/1954 )

Reply:
K thanks.  I got that from https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=channel_sources if anyone here wants to update that

Reply:
Thanks for pointing that out. @jaimergp just updated that here ( https://github.com/conda-forge/conda-forge.github.io/pull/1613 )